Entity/CategoryManager.php

```diff
@@ -136,6 +176,31 @@ public function getRootCategories($loadChildren = true)
     }
 
     /**
+     * @param bool|true $loadChildren
+     *
+     * @return array
+     */
+    public function getRootCategoriesSplitByContexts($loadChildren = true)
```

Comment:
@core23, Yes.
Also we probably need to remove `getRootCategory` and `getRootCategories` methods.
At the current moment they are used in MediaBundle and `AbstractCategoriesBlockService`, so we need them for BC.
I added `@todo NEXT_MAJOR` comments for them.